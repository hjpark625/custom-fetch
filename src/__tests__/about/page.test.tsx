import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import AboutPage from '@/app/about/page';

describe('page', () => {
  it('renders a heading', () => {
    render(<AboutPage />);

    const heading = screen.getByRole('heading', { level: 1 });
    const testContainer = screen.getByTestId('custom-element');

    expect(heading).toBeInTheDocument();
    expect(testContainer).toBeInTheDocument();
    // expect(heading).toHaveTextContent('welcome to About');
  });

  it('renders homepage unchanged', () => {
    const { container } = render(<AboutPage />);
    expect(container).toMatchSnapshot();
  });
});
