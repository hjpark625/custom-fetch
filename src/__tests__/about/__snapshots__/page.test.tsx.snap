// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page renders homepage unchanged 1`] = `
<div>
  <h1
    data-testid="custom-element"
  >
    welcome to About
  </h1>
</div>
`;
